Read & Return Ebook buttons sep internetarchive#2972:v1

Aj-KamaL opened the pull request:
<!-- What issue does this PR close? -->
Closes #2972
The issue was more general and not limited to read and return buttons.(borrow, join_waitlist)  
Also, one of the form tags was not closed. The PR adds a top-margin: 5px to all .cta-buttons to add space.

### Technical
<!-- What should be noted about the implementation? -->
```css
.cta-button-group {
....
  .cta-btn {
    margin:0;
    ...}
```
The above CSS snippets set all margins to 0 including the margin-top: 5px. I set a default margin-top: 5px for all cta-buttons, now Read and Listen are part of .cta-button-group hence their top margin will be set 0, but other buttons will have a margin of 5px at the top like Return.

### Testing
<!-- Steps for reviewer to reproduce/verify what this PR does/fixes. -->
In the web version, the error only appears to exist in edition pages:/works only but in smartphones, the error exists in the edition tools panel: /books/ of a borrowed book.

### Evidence
<!-- If this PR touches UI, please post evidence (screenshots) of it behaving correctly. -->
Before:
![image](https://user-images.githubusercontent.com/23471306/76911117-ef47bc00-68d5-11ea-9886-4df53400ad61.png)


After:
![image](https://user-images.githubusercontent.com/23471306/76911133-fcfd4180-68d5-11ea-8e60-dbb20fc08a54.png)

### Stakeholders
<!-- @ tag stakeholders of this bug -->

ishank-dev commented on openlibrary/macros/LoanStatus.html:
```diff
@@ -38,10 +38,13 @@
   $:macros.ReadButton(ocaid, loan=user_loan, listen=True)
   $ return_url = page.url().rsplit('/', 1)[0] + '/do_return/borrow'
   <form action="$return_url" method="post" class="waitinglist-form return-book">
+    <p>
```
Why is <p> tag even required?

Aj-KamaL replied:
The above read cta_btn_grp occurs in pairs and forms a segment of their own. Any next button(ret in this case) that comes immediately next to it collides into it based on the available space and missing line. So even if something that looks fine in Desktop version(a) looks collided in the mobile version(b).

![a](https://user-images.githubusercontent.com/23471306/76710253-f763f800-672b-11ea-90d2-cd4fa4c37776.png)
![b](https://user-images.githubusercontent.com/23471306/76710254-f8952500-672b-11ea-9430-50159afedfeb.png)

cdrini commented:
Hi @Aj-KamaL ! Thanks for the PR. Please don't forget to fill in the template of the PR; specifically the steps you took for testing, a screenshot of the result (or even better, a before/after), and the issue number will help us better/more quickly review the change.  I've added the template as a stub for you :+1:

Aj-KamaL commented:
> Hi @Aj-KamaL ! Thanks for the PR. Please don't forget to fill in the template of the PR; specifically the steps you took for testing, a screenshot of the result (or even better, a before/after), and the issue number will help us better/more quickly review the change. I've added the template as a stub for you

Noted! Thanks for sharing the template. I have updated the PR's comment accordingly.
Let me know if the changes resolved the issue or it still persists.
